src/Entity/TranslationRequest.php

```diff
@@ -160,101 +161,29 @@ public function setTargetLanguageCodes(array $target_language_codes): Translatio
   /**
    * {@inheritdoc}
    */
-  public function getJobs(): array {
-    return $this->get('jobs')->referencedEntities();
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function getJobIds(): array {
-    return $this->get('jobs')->isEmpty() ? [] : array_column($this->get('jobs')->getValue(), 'target_id');
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function hasJob(string $job_id): bool {
-    return in_array($job_id, $this->getJobIds());
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function addJob(string $job_id): TranslationRequestInterface {
-    $jobs = $this->getJobIds();
-    $jobs[] = $job_id;
-    return $this->set('jobs', array_unique($jobs));
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function autoAcceptsTranslations(): bool {
-    return (bool) $this->get('auto_accept_translations')->value;
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function setAutoAcceptTranslations(bool $value): TranslationRequestInterface {
-    return $this->set('auto_accept_translations', $value);
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function getTranslationSync(): array {
-    return $this->get('translation_synchronisation')->first()->getValue();
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function setTranslationSync(array $translation_synchronisation): TranslationRequestInterface {
-    return $this->set('translation_synchronisation', $translation_synchronisation);
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function upstreamsTranslations(): bool {
-    return (bool) $this->get('upstream_translations')->value;
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function setUpstreamTranslation(bool $value): TranslationRequestInterface {
-    return $this->set('upstream_translations', $value);
-  }
-
-  /**
-   * {@inheritdoc}
-   */
-  public function getMessage(): ?string {
-    return $this->get('message')->value;
+  public function getRequestStatus(): string {
+    return $this->get('request_status')->value;
   }
 
   /**
    * {@inheritdoc}
    */
-  public function setMessage(string $message): TranslationRequestInterface {
-    return $this->set('message', $message);
+  public function setRequestStatus(string $request_status): TranslationRequestInterface {
+    return $this->set('request_status', $request_status);
   }
 
   /**
    * {@inheritdoc}
    */
-  public function getRequestStatus(): string {
-    return $this->get('request_status')->value;
+  public function getData(): string {
+    return Json::decode($this->get('data')->value);
   }
 
   /**
    * {@inheritdoc}
    */
-  public function setRequestStatus(string $request_status): TranslationRequestInterface {
-    return $this->set('request_status', $request_status);
+  public function setData(string $data): TranslationRequestInterface {
```

Comment:
Well, if you encode into a json string, then we need to pass an array here no? Not a string...

This shows also that this test is useless:

```
    $data_for_translation = $this->randomString();
    $translation_request->setData($data_for_translation);
    $this->assertEquals($data_for_translation, $translation_request->getData());
```